docs/user/otp-deployment.md

```diff
@@ -0,0 +1,14 @@
+# OTP Deployment Guide
+
+## OTP Deployment Architecture
```

Comment:
I feel like since this is the sort of the landing page for OTP deployments that it should have a paragraph or two that introduces the overall architecture.